src/common/createplay/CreatePlay.jsx

```diff
@@ -121,7 +121,7 @@ const CreatePlay = () => {
     if (refreshToken) {
       console.log(refreshToken);
     } else {
-      window.location = NHOST.AUTH_URL;
+      window.location = NHOST.AUTH_URL('http://localhost:3000/plays/create');
```

Comment:
Also, it is not guaranteed that the dev will always run on the 3000 port. It should be from the environment variable and in code default to 3000

Reply:
ok will fix this